recipes/qt/5.x.x/conanfile.py

```diff
@@ -1077,6 +1077,7 @@ def _create_plugin(pluginname, libname, type, requires):
 
         if self.options.qtwinextras:
             _create_module("WinExtras")
+            self.cpp_info.components["qtWinExtras"].system_libs.append("dwmapi")
```

Comment:
Is `qtwinextras` a valid option on non-Windows systems?
As far as I can see, it's not removed in `config_options`.

Reply:
No, it's not a valid option for non windows platforms, [see here] (https://doc.qt.io/qt-5/qtwinextras-index.html)